src/apps/service/apps/agave/jobs.clj

```diff
@@ -119,6 +119,7 @@
     :batch           false
     :description     (:description job)
     :enddate         (:enddate job)
+    :system_id       "agave"
```

Comment:
thanks for catching this. I meant to use a separate constant here rather than the value from `mescal.agave-de-v2.constants`